Dockerfile

```diff
@@ -33,7 +33,7 @@ ARG CRI_TOOLS_VERSION=v1.30.0
 # Legacy builder that doesn't support TARGETARCH should set this explicitly using --build-arg.
 # If TARGETARCH isn't supported by the builder, the default value is "amd64".
 
-FROM golang:1.21.5-bullseye AS golang-base
+FROM golang:1.22.3-bullseye AS golang-base
```

Comment:
On ` runc-dev` stage (line 81), please use go 1.21 to fix the current CI failure (`OCI runtime create failed: runc create failed: unable to start container process: error during container init: read init-p: connection reset by peer: unknown`).